java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -897,6 +897,53 @@ public void sismember(BaseClient client) {
         assertTrue(executionException.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void sinterstore(BaseClient client) {
+        String key1 = "{key}-1-" + UUID.randomUUID();
+        String key2 = "{key}-2-" + UUID.randomUUID();
+        String key3 = "{key}-3-" + UUID.randomUUID();
+        String key4 = "{key}-4-" + UUID.randomUUID();
+        String key5 = "{key}-5-" + UUID.randomUUID();
+
+        assertEquals(3, client.sadd(key1, new String[] {"a", "b", "c"}).get());
+        assertEquals(3, client.sadd(key2, new String[] {"c", "d", "e"}).get());
+        assertEquals(3, client.sadd(key4, new String[] {"e", "f", "g"}).get());
+
+        // create new
+        assertEquals(1, client.sinterstore(key3, new String[] {key1, key2}).get());
+        assertEquals(Set.of("c"), client.smembers(key3).get());
+
+        // overwrite existing set
+        assertEquals(1, client.sinterstore(key2, new String[] {key3, key2}).get());
+        assertEquals(Set.of("c"), client.smembers(key2).get());
+
+        // overwrite source
```

Comment:
I don't think testing overwrites add any value but I am not against it. Everything looks fine approved.